serverless/vscodeExtensionFeedbackSubmit.functions/submit.js

```diff
@@ -0,0 +1,48 @@
+const axios = require('axios');
+const {
+  VSCODE_EXTENSION_FEEDBACK_ACCESS_TOKEN,
+  VSCODE_EXTENSION_FEEDBACK_HUBDB_TABLE_NAME,
+} = process.env;
+const HUBDB_API_PATH = `https://api.hubspot.com/cms/v3/hubdb`;
+
+const addTableRow = async (rowData) => {
+  return axios({
+    method: 'POST',
+    url: `${HUBDB_API_PATH}/tables/${VSCODE_EXTENSION_FEEDBACK_HUBDB_TABLE_NAME}/rows`,
+    data: {
+      values: rowData,
+    },
+    headers: {
+      authorization: `Bearer ${VSCODE_EXTENSION_FEEDBACK_ACCESS_TOKEN}`,
+    },
+  });
+};
+
+const publishTable = async () => {
+  return axios({
+    method: 'POST',
+    url: `${HUBDB_API_PATH}/tables/${VSCODE_EXTENSION_FEEDBACK_HUBDB_TABLE_NAME}/draft/push-live`,
+    data: {},
+    headers: {
+      authorization: `Bearer ${VSCODE_EXTENSION_FEEDBACK_ACCESS_TOKEN}`,
+    },
+  });
+};
+
+exports.main = async (context, sendResponse) => {
```

Comment:
With this being public especially, is there any concern about malicious behavior here? Is there any protection against someone spamming it with junk data? I'm assuming the the hubdb API already cleansing inputs, not familiar with what security serverless functions offer out of the box.

Reply:
Yes, definitely is a concern, but I think that is an issue that we deal with elsewhere as well. The endpoint to add the row to hubdb should be able to handle strings that are too long, etc.

I added some `maxlength`s to the inputs. If you have any other ideas of what else can be done, happy to add.